docker-compose-services/php8_1/README.md

```diff
@@ -0,0 +1,29 @@
+# PHP8.1 (while in alpha/beta/etc)
+
+DDEV-Local will directly support PHP8 as soon as possible, but not likely until the full release in late 2020. At that time this recipe will be obsolete.
+
+In the meantime, you can use the [official docker php image](https://hub.docker.com/_/php) (currently php:8.0.0alpha3-fpm-buster) to serve PHP.  This can be updated as new versions come out.
```

Comment:
```suggestion
In the meantime, you can use the [official docker php image](https://hub.docker.com/_/php) (currently php:8.1.0alpha1-fpm-buster) to serve PHP.  This can be updated as new versions come out.
```